multicluster/controllers/multicluster/commonarea/labelidentityimport_controller_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2022 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commonarea
+
+import (
+	"reflect"
+	"testing"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	mcsv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/multicluster/controllers/multicluster/common"
+)
+
+var (
+	labelIdentityResImportName = "label-identity-app-client"
+
+	labelIdentityResImpReq = ctrl.Request{NamespacedName: types.NamespacedName{
+		Name: labelIdentityResImportName,
+	}}
+
+	labelIdentityResImport = &mcsv1alpha1.ResourceImport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: labelIdentityResImportName,
+		},
+		Spec: mcsv1alpha1.ResourceImportSpec{
+			LabelIdentity: &mcsv1alpha1.LabelIdentitySpec{
+				Label: "namespace:kubernetes.io/metadata.name=ns&pod:app=client",
+				ID:    uint32(1),
+			},
+		},
+	}
+)
+
+func TestLabelIdentityResourceImportReconciler_handleLabelIdentityCreateEvent(t *testing.T) {
+	remoteMgr := NewRemoteCommonAreaManager("test-clusterset", common.ClusterID(localClusterID), "kube-system")
```

Comment:
Hi @GraysonWu  there are a few common area changes in PR #3964, this method is removed. Could you rebase to latest codes and refine the codes? 
Btw, there are a few github action check failure, please fix them.

Reply:
Thanks for the reminder. I'm still in dev testing phase so I haven't updated those test files. Later I will address it and all your comments below together.

Reply:
Test codes refined.